### PR TITLE
tus client py change to 1 step upload

### PR DESCRIPTION
**Pull request by cosmycx**

### CODE REVIEW

File: tus-clients/python/tuspy-uploader-test-ndlp-ri-good.py [hunk 1]
```diff
@@ -1,38 +1,66 @@
...
-# upload the entire file
-try:
-    uploader.upload()
-except TusCommunicationError as error:
-    print('TusCommunicationError: ' + str(error.response_content.decode('UTF-8')))
+ACCOUNT_USERNAME = os.getenv('ACCOUNT_USERNAME')
+ACCOUNT_PASSWORD = os.getenv('ACCOUNT_PASSWORD')
+TUS_UPLOAD_URL = os.getenv('DEX_URL')
+
+if ACCOUNT_USERNAME == "" or ACCOUNT_PASSWORD == "" or TUS_UPLOAD_URL == "":
```
Comment (boris-ning-usds) [inline tus-clients/python/tuspy-uploader-test-ndlp-ri-good.py:15]: Can you support check all three environment variables and accumulatively tell me which one is blank? ex. "Error: ACCOUNT_PASSWORD and TUS_UPLOAD_URL environment variable is blank."

**Review by boris-ning-usds — CHANGES_REQUESTED** [review]
Comments added.

File: tus-clients/python/tuspy-uploader-test-ndlp-ri-good.py (hunk 1 above)
Comment (cosmycx) [reply tus-clients/python/tuspy-uploader-test-ndlp-ri-good.py]: changed as requested

**Review by boris-ning-usds — APPROVED** [review]
LGTM!

**Pull request merged**